tests/Ukraine/UkraineTest.php

```diff
@@ -35,7 +36,7 @@ public function testOfficialHolidays(): void
         $this->assertDefinedHolidays([
             'newYearsDay',
             'internationalWorkersDay',
-            'secondInternationalWorkersDay',
+            //'secondInternationalWorkersDay',  // until 2018
```

Comment:
Please also use a condition here rather than just removing the holiday. You may want to look at `/tests/Germany/Saxony/SaxonyTest.php` for some inspiration.

Reply:
adjusted like `/tests/Germany/Saxony/SaxonyTest.php`